src/services/disclaimer.js

```diff
@@ -0,0 +1,153 @@
+goog.provide('ngeo.Disclaimer');
+
+goog.require('goog.asserts');
+goog.require('ngeo');
+goog.require('ngeo.Message');
+
+
+/**
+ * Provides methods to display any sort of messages, disclaimers, errors,
+ * etc. Requires Bootstrap library (both CSS and JS) to display the alerts
+ * properly.
+ *
+ * @constructor
+ * @extends {ngeo.Message}
+ * @ngdoc service
+ * @ngname ngeoDisclaimer
+ * @ngInject
+ */
+ngeo.Disclaimer = function() {
+
+  goog.base(this);
+
+  var container = angular.element('<div class="ngeo-disclaimer"></div>');
```

Comment:
@pgiraud: I used `"ngeo-disclaimer"` instead of `"disclaimer"`.  I recommend using the extra namespace, here.